drake_ros_core/package.xml

```diff
@@ -24,5 +32,7 @@
 
   <export>
     <build_type>ament_cmake</build_type>
+    <buildtool_export_depend>eigen3_cmake_module</buildtool_export_depend>
+    <build_export_depend>eigen</build_export_depend>
```

Comment:
These lines actually belong on the same level as the other `<depend>` lines. They don't do anything inside an `<export>` block.